public/app/features/explore/Logs/LogsColumnSearch.tsx

```diff
@@ -0,0 +1,23 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data/src';
+import { Field, Input, useTheme2 } from '@grafana/ui/src';
+
+function getStyles(theme: GrafanaTheme2) {
+  return {
+    searchWrap: css({
+      padding: theme.spacing(0.4),
+    }),
+  };
+}
+
+export const LogsColumnSearch = (props: { onChange: (e: React.FormEvent<HTMLInputElement>) => void }) => {
```

Comment:
Just a NIT: https://github.com/grafana/grafana/blob/main/contribute/style-guides/frontend.md#how-to-declare-functional-components